lib/ui/map/location_detail_sheet.dart

```diff
@@ -175,8 +184,8 @@ class _ChangeDateButton extends StatelessWidget {
         Future<DateTime> selectedDate = showDatePicker(
           context: context,
           initialDate: this.selectedDate,
-          firstDate: DateTime.now().subtract(Duration(days: 1)),
```

Comment:
I was not able to reproduce your bug @Johannes11833 but I strongly suspect that this part caused the issue.